packages/sdk-client-v3/src/types/types.d.ts

```diff
@@ -242,6 +242,7 @@ export type RetryMiddlewareOptions = {
   maxRetries?: number
   retryDelay?: number
   maxDelay?: typeof Infinity
+  retryOnAbort?: boolean
```

Comment:
Please check the docs, I could see this was removed in the v3 docs [here](https://github.com/commercetools/commercetools-docs/pull/4986/files#diff-4c9187ddfa33195642947e7508f80e1fced9d8e01127e80d1c4b2d9544f46524L16)

Reply:
https://github.com/commercetools/commercetools-docs/pull/4986/commits/d763b02415c2bf6a4b13f27f38f3a12bd7b241fa